Sources/KeyboardShortcuts/Localization/en.lproj/Localizable.strings

```diff
@@ -3,3 +3,5 @@
 "keyboard_shortcut_used_by_menu_item" = "This keyboard shortcut cannot be used as it’s already used by the “%@” menu item.";
 "keyboard_shortcut_used_by_system" = "This keyboard shortcut cannot be used as it’s already a system-wide keyboard shortcut.";
 "keyboard_shortcuts_can_be_changed" = "Most system-wide keyboard shortcuts can be changed in “System Settings › Keyboard › Keyboard Shortcuts”.";
+"force_use_shortcut" = "Use Anyway";
+"ok" = "OK";
```

Comment:
I'm not very familiar with localisations in Swift/Xcode projects, please let me know if there's a built-in default for "OK"/"yes" dialogs provided by the localisation framework; I'm happy to use that if there is 🙂

Reply:
There's nothing built-in.

Would be great if you could fill in "OK" for the existing localizations though: https://applelocalization.com/macos?q=OK&l=English&l=French&l=German&l=Italian&l=Japanese&l=Spanish

Reply:
Cool, I've added them for all languages in the repo, based on the text in Apple's docs. Thanks! 🙂